public/app/themes/justice/src/patch/js/ccfw-gtm.js

```diff
@@ -57,167 +69,189 @@ const CCFW = {
    */
     trackEvent: (event, object) => {
         if (!object) {
-            dataLayer.push({ 'event': event });
-            return;
+            dataLayer.push({ 'event': event })
+            return
         }
-        dataLayer.push(jQuery.extend({}, { 'event': event }, object));
-    },
-  /**
-   * Determine exactly what element was clicked and return its jQuery equivalent
-   * @param event
-   * @returns {jQuery|HTMLElement}
-   * @private
-   */
-    jqTarget: function (event) {
-        return jQuery(event.target);
+        dataLayer.push(jQuery.extend({}, { 'event': event }, object))
     },
   /**
    * Runs on load
    */
-    clearStorage: () => {
-        let stored = CCFW.storage.time.get(); // always a year from storage
-        let now = new Date().getTime();
+    maybeExpired: () => {
+        let stored = CCFW.storage.time.get() // always a year from storage
+        let now = new Date().getTime()
 
         if (now > stored) { // a year has past
-            CCFW.storage.clear('ccfw-gtm-allowed');
-            CCFW.storage.clear('ccfw-banner-hidden');
-            CCFW.storage.clear('ccfw-time');
+            CCFW.storage.clear('ccfw-gtm-allowed')
+            CCFW.storage.clear('ccfw-banner-hidden')
+            CCFW.storage.clear('ccfw-time')
         }
     },
 
   /**
    * @param remove acknowledges that we are removing all allowed ids
    */
     toggleAll: function (remove) {
-        let allowList = CCFW.storage.allowed.get() || [];
+        let allowList = CCFW.storage.allowed.get() || []
 
         if (remove) {
-            allowList = [];
+            allowList = []
         }
 
-        jQuery('.' + CCFW.selector.toggles).each(function (key, element) {
-            let allowed = jQuery(element).data('allowlist');
+        CCFW.jq.toggles.each(function (key, element) {
+            let allowed = jQuery(element).data('allowlist')
```

Comment:
I'm not sure, but as toggles is a jQuery object, does element need to be wrapped in jQuery() here?

Reply:
Element is actually a DOM object ☺️ we convert it here

Reply:
https://api.jquery.com/Types/#Element

Reply:
For reference:
https://api.jquery.com/each/#each-function

```javascript
$( "div" ).each(function( index, element ) {
    // element == this ... a document object model DIV element
    $( element ).css( "backgroundColor", "yellow" );
    if ( $( this ).is( "#stop" ) ) {
      $( "span" ).text( "Stopped at div index #" + index );
      return false;
    }
  });
```

Reply:
Ah, I see, thanks for explaining :D